drivers/wasapi/audio_driver_wasapi.cpp

```diff
@@ -206,21 +201,7 @@ class CMMNotificationClient : public IMMNotificationClient {
 
 static CMMNotificationClient notif_client;
 
-typedef const char *(CDECL *PWineGetVersionPtr)(void);
-
-bool AudioDriverWASAPI::is_running_on_wine() {
-	HMODULE nt_lib = LoadLibraryW(L"ntdll.dll");
-	if (!nt_lib) {
-		return false;
-	}
-
-	PWineGetVersionPtr wine_get_version = (PWineGetVersionPtr)GetProcAddress(nt_lib, "wine_get_version");
-	FreeLibrary(nt_lib);
-
-	return (bool)wine_get_version;
-}
-
-Error AudioDriverWASAPI::audio_device_init(AudioDeviceWASAPI *p_device, bool p_capture, bool reinit) {
+Error AudioDriverWASAPI::audio_device_init(AudioDeviceWASAPI *p_device, bool p_capture, bool reinit, bool no_audio_client_3) {
```

Comment:
Nitpick, but parameters should start with `p_` (and `reinit` was also not properly following this convention, would be worth fixing too while at it).